### ISSUE TITLE
Washington Post links do not work

### ISSUE THREAD

**Issue by tasomaniac** [tasomaniac/OpenLinkWith#71]
https://www.washingtonpost.com/sports/2020/07/29/rudy-gobert-coronavirus-nba-return/

**Comment by tcely** [comment]
It looks like these lines should just be removed. What was the intention here anyway?

A Washington Post app should register links to open, not depend on the scheme being changed.

https://github.com/tasomaniac/OpenLinkWith/blob/8785eb3
67fabf9a44d05de44cb9997ffd1462c85/redirect/src/main/kotlin/com/tasomaniac/openwith/redirect/UrlFix.kt#L151-L157

https://github.com/tasomaniac/OpenLinkWith/blob/8785eb367fabf9a44d05de44cb9997ffd1462c85/redirect/src/main/kotlin/com/tasomaniac/openwith/redirect/UrlFix.kt#L167

If you could let me know if this is an acceptable solution before I create a PR for this, I would appreciate it.

**Comment by tasomaniac** [comment]
Yeah. I think that's it. I was gonna remove it but then I tested it on my phone and it was suddenly working fine. That's why I didn't do it. If you verify that it works fine without these, we should just remove it.

The reason was that Washington Post app did not even support their own links. When I added that, it was couple years ago. It could be that they fixed their app which made my integration broken.

**Comment by tcely** [comment]
I can confirm it is currently broken when the Washington Post app is not installed.

I follow them on Twitter and was confused why only their links wouldn't work. I finally got tired of sharing the tweet to another browser to open the article and tracked down this issue.

Let's try removing it and if their app is still broken we might add permissions that allow checking if it is installed before doing this rewriting. Or just make it a setting the user can toggle, as that is a better privacy approach.

**Comment by tasomaniac** [comment]
I see. It is broken when the app is not installed.
I designed this app to be used with native apps installed. With the feature that makes OLW acts like a browser, things get more complex actually.

It's more clear now. I will look into this soon-ish and see what I can do.

If you want to open a Pull Request disabling Washington Post specific part, please go ahead, I can release a version with that change much quicker.

**Comment by tcely** [comment]
Ok. The pull request removing the Washington Post rewriting is submitted.

**Comment by tcely** [comment]
As an aside. Since Android has made it harder to not have a default browser configured in recent versions, I find this app really useful for choosing which browser to use for links.

For example, Google products can use Chrome and always be logged in, and news sites can use a browser with better privacy settings, etc.

**Comment by bengalih** [comment]
Thanks for this fix, I can also confirm it works.
Please release an official build.
